src/pages/signin/SignInPageLayout/Footer.tsx

```diff
@@ -1,33 +1,37 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
-import _ from 'underscore';
 import SignInGradient from '@assets/images/home-fade-gradient--mobile.svg';
 import Hoverable from '@components/Hoverable';
 import * as Expensicons from '@components/Icon/Expensicons';
 import ImageSVG from '@components/ImageSVG';
 import Text from '@components/Text';
 import TextLink from '@components/TextLink';
-import withLocalize, {withLocalizePropTypes} from '@components/withLocalize';
+import useLocalize from '@hooks/useLocalize';
 import useStyleUtils from '@hooks/useStyleUtils';
 import useTheme from '@hooks/useTheme';
 import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
 import Licenses from '@pages/signin/Licenses';
 import Socials from '@pages/signin/Socials';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
+import type {TranslationPaths} from '@src/languages/types';
 
-const propTypes = {
-    ...withLocalizePropTypes,
-    navigateFocus: PropTypes.func.isRequired,
-    shouldShowSmallScreen: PropTypes.bool,
+type FooterProps = {
+    navigateFocus: () => void;
+    shouldShowSmallScreen?: boolean;
 };
 
-const defaultProps = {
-    shouldShowSmallScreen: false,
+type FooterColumnRow = ({onPress: () => void; link?: string} | {onPress?: () => void; link: string}) & {
```

Comment:
```suggestion
type FooterColumnRow = (LinkProps | PressProps) & {
```

Import LinkProps and PressProps from TextLink